label_studio/ui_test/ui_test_label_studio.py

```diff
@@ -0,0 +1,435 @@
+#
+# Author: Hovhannes Atoyan (hovhannes.atoyan@gmail.com)
+# UI automation for Label Studio
+# Automated base steps for Image labeling (Setup, Import, Start, Export)
+#
+
+import os
+import time
+import datetime
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.firefox.firefox_binary import FirefoxBinary
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.action_chains import ActionChains
+
+
+class myTestCase(unittest.TestCase):
+    def setUp(self):
+        # self.driver = webdriver.Firefox()
+        #
+        self.downloadDir = './'
+        fp = webdriver.FirefoxProfile()
+        fp.set_preference("browser.download.folderList", 2)
+        fp.set_preference("browser.download.manager.showWhenStarting", False)
+        fp.set_preference("browser.download.dir", self.downloadDir)
+        # fp.set_preference("browser.helperApps.neverAsk.saveToDisk", "attachment/csv")
+        # fp.set_preference("browser.helperApps.neverAsk.saveToDisk", "attachment/json")
+        fp.set_preference("browser.helperApps.neverAsk.saveToDisk", "attachment/zip")
+        self.driver = webdriver.Firefox(firefox_profile=fp)
+        self.wait = WebDriverWait(self.driver, 10)
+        #
+        self.site_url = 'http://localhost:8080/welcome'
+        # self.current_page = self.site_url
+        self.site_logo = '//a[@id="logo"]/img'
+        self.page_import_url = 'http://localhost:8080/import'
+        #
+        # MENU
```

Comment:
how to customize all those elements when xpaths are changed on page? could you please provide some instructions in comments/readme

Reply:
All elements (from each page) defined into function   "def setUp(self)"
(xpathes, css-selectors, iIDs,...  all are here)
Each element is variable, which is used into test-script.
So, when xpath will changes, we only need to do changing only in one place (into function   "def setUp(self)" )
Sure, I will add more detailed instructions into readme file(on my next commit)